coderd/database/queries/quotas.sql

```diff
@@ -28,6 +28,9 @@ FROM
 INNER JOIN
 	workspaces on wb.workspace_id = workspaces.id
 WHERE
+	-- Only return workspaces that match the user + organization.
+	-- Quotas are calculated per user per organization.
+	NOT workspaces.deleted AND
```

Comment:
This `workspaces.deleted` being placed in this join sub query prevents the workspaces `seq scan` from what I can tell.

The alternative place is to place it in the final query as `NOT workspaces.deleted`.

----

This is equivalent though because we are doing an `INNER JOIN` on `workspace_builds` and `workspaces`. If the `workspace` is deleted, it's builds are completely omitted from `latest_builds`